examples/example-gateway/config/production.json

```diff
@@ -1,15 +1,18 @@
 {
 	"clients.bar.ip": "127.0.0.1",
 	"clients.bar.port": 4001,
+	"clients.bar.timeout": 1000,
```

Comment:
nitpick: 10000; 10 seconds is a better default thats less aggressive.

Reply:
Done